server/file.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+const remark = require('remark');
+const lint = require('remark-lint');
+const html = require('remark-html');
+
+function getMarkdown(fileName) {
+  return new Promise((resolve, reject) => {
+    fs.readFile(fileName, (err, markdown) => {
+      if (!err) {
+        resolve(markdown.toString('utf-8'));
+      } else {
+        reject(err.message);
+      }
+    });
+  });
+}
+
+function getFileContent() {
+  return new Promise((resolve, reject) => {
+    getMarkdown('./TEST.md').then((markdown) => {
+      remark().use(lint).use(html).process(markdown, function (err, markdownHtml) {
```

Comment:
1. About lint - https://github.com/wooorm/remark-lint  :)
2. I think processing to html on server could be appropriate choice for view of md files, therefore, in my opinion, it's ok for now, for MVP. 
But in case if we will need functionality of editing of md files, abstract tree will be more useful.
So, as I understood, because of mdast is a part of remark, we can easily use mdast for generating such trees for client in future.